### PR TITLE
Update zh tw translation

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by jserv** [comment]
Please rework your commits.

**Comment by PeterDaveHello** [comment]
@jserv updated! Thank you.

File: ts/chewing-editor_zh_TW.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@
     </message>
     <message>
         <source>Import %1 successful.</source>
-        <translation type="vanished">匯入 &quot;%1&quot; 成功。</translation>
+        <translation type="vanished">成功 &quot;%1&quot; 成功。</translation>
```
Comment (david50407) [inline ts/chewing-editor_zh_TW.ts]: 成功 for what?

Comment (PeterDaveHello) [inline ts/chewing-editor_zh_TW.ts]: Oh that's a mistake after fixing the conflicts, thanks.

**Comment by jserv** [comment]
Please use `git rebase -i` to submit single-commit PR since we would only defer to.

**Comment by PeterDaveHello** [comment]
Hi @jserv , I use two commits since there is a little bit difference in that two commit. I updated to a single commit now.

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**